setup.cfg

```diff
@@ -0,0 +1,36 @@
+[metadata]
+name = pytest-marker-bugzilla
+author = Eric Sammons
+author-email = elsammons@gmail.com
+summary = py.test bugzilla integration plugin, using markers
+description-file = README.md
+home-page = https://github.com/eanxgeek/pytest_marker_bugzilla.git
+license = GPL2
+zip_safe=False
+classifier =
+    Development Status :: 3 - Alpha
+    Topic :: Utilities
+    License :: OSI Approved :: GNU General Public License 2 (GPL2),
+    Operating System :: POSIX,
+    Operating System :: Microsoft :: Windows,
+    Operating System :: MacOS :: MacOS X,
+    Topic :: Software Development :: Testing,
+    Topic :: Software Development :: Quality Assurance,
+    Topic :: Utilities,
+    Programming Language :: Python,
+    Programming Language :: Python :: 2.6,
+    Programming Language :: Python :: 2.7,
+    Programming Language :: Python :: 3.4,
+keywords =
+    pytest
+    bugzilla
+    plugin
+[files]
+packages =
+    pytest_marker_bugzilla
+[bdist_rpm]
+requires =
+    pytest >= '2.2.4'
+    python-bugzilla >= '0.6.2'
+    pbr >= '0.8.2'
+    six
```

Comment:
you are missing entry_points .... that is why tests failed, because plugin is not visible for pytest ...